chrome-extension/popup.js

```diff
@@ -0,0 +1,90 @@
+const API_KEY = 'AIzaSyAed5yuxCNrXlEFAqSoWwAQpE3Ng95Tzl8';
```

Comment:
Would be a good idea to hide the API_KEY in another config file and .gitignore it. Once you import here, you should be able to access the API_KEY as config.API_KEY or something like that.